sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/_tests_/e2e/student-scholastic-standings.institutions.controller.saveScholasticStanding.e2e-spec.ts

```diff
@@ -253,7 +261,147 @@ describe("StudentScholasticStandingsInstitutionsController(e2e)-saveScholasticSt
     ).toBe(createdScholasticStandingId);
   });
 
-  it("Should create a new scholastic standing for a part-time application when the institution user requests.", async () => {
+  it("Should create a new scholastic standing School transfer for a part-time application when the institution user requests.", async () => {
+    // Arrange
+    mockFormioDryRun({
+      studentScholasticStandingChangeType:
+        StudentScholasticStandingChangeType.SchoolTransfer,
+    });
```

Comment:
Minor. The mocking can be after the data is prepared. 

It would be good to  add a todo comment to enhance this test when the form.io e2e container is in place. This is mainly because the logic validation of dates is happening in form.io.